hbase-server/src/main/java/org/apache/hadoop/hbase/snapshot/CreateSnapshot.java

```diff
@@ -67,24 +67,14 @@ protected void processOptions(CommandLine cmd) {
 
     @Override
     protected int doWork() throws Exception {
-        Connection connection = null;
-        Admin admin = null;
-        try {
-            connection = ConnectionFactory.createConnection(getConf());
-            admin = connection.getAdmin();
+        try (Connection connection = ConnectionFactory.createConnection(getConf())) {
```

Comment:
A syntax sugar, but you can do try with multiple resources 
`
try (Connection connection = ConnectionFactory.createConnection(getConf());
            Admin admin = connection.getAdmin()) {
}
`

Reply:
Thank you :) I modified the code based on your suggestion.